phalcon/db/adapter/pdo/mysql.zep

```diff
@@ -280,7 +280,10 @@ class Mysql extends PdoAdapter implements AdapterInterface
 			 */
 			if memstr(columnType, "(") {
 				let matches = null;
-				if preg_match(sizePattern, columnType, matches) {
+
+				if definition["type"] == Column::TYPE_ENUM {
+					let definition["size"] = substr(columnType, 5, -1);
```

Comment:
@sergeyklay this is where we were trying to use a regex but its easier to simply get the contents out of enum(CONTENTS) by doing a substr select. 

So essentially if you have ENUM('val1', 'val2', 'val3') then definition["size"] = "'val1', 'val2', 'val3'"

Does that make more sense?